generator/src/main/java/org/corfudb/generator/LongevityRun.java

```diff
@@ -0,0 +1,83 @@
+package org.corfudb.generator;
+
+import java.time.Duration;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * Created by rmichoud on 7/27/17.
+ */
+
+/**
+ * This longevity test launcher will set the duration of the test
+ * based on inputs.
+ */
+public class LongevityRun {
+    public static void main(String[] args) {
+        long longevity;
+
+        Options options = new Options();
+
+        Option amountTime = new Option("t", "time_amount", true, "time amount");
+        amountTime.setRequired(true);
+        Option timeUnit = new Option("u", "time_unit", true, "time unit (s, m, h)");
+        timeUnit.setRequired(true);
+        Option corfuEndpoint = new Option("c", "corfu_endpoint", true,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "corfu_endpoint" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)